mender-auth/cli/cli.cpp

```diff
@@ -130,6 +130,21 @@ error::Error DoMain(
 	return action.value()->Execute(context);
 }
 
+int Main(const vector<string> &args, function<void(context::MenderContext &ctx)> test_hook) {
```

Comment:
Commit message is a bit funky. Did you mean to write `mender::client::one` as a namespace?

Reply:
Hahaha. I meant to write ... `mender:update:cli` with `mender:auth`'s one (as in its `:cli`).

I'll rewrite.